core/cfn/build_demos_prod.yml

```diff
@@ -34,6 +34,12 @@ Parameters:
     Description: >-
         The compute type. i.e. BUILD_GENERAL1_LARGE (15Gb RAM, 8vCPU) or BUILD_GENERAL1_2XLARGE (145 Gb RAM, 72 vCPU)
     Type: String
+
+  GithubSourceOrg:
```

Comment:
I've added this to [checklayer](https://github.com/aws/meta-aws-ci/blob/master/core/cfn/ci_checklayer.yml#L18) also. We should update the name to be consistent.